### ISSUE TITLE
Caffe to CCV convnet import script?

### ISSUE THREAD

**Issue by geraldchao** [liuliu/ccv#121]
Is there any interest in having a script that converts/imports pre-trained models from Caffe into CCV? Examples of these models are VGG's 2014 models and ones from MIT (https://github.com/BVLC/caffe/wiki/Model-Zoo). There's also an existing python script for importing into vlfeat here: https://github.com/vlfeat/matconvnet/blob/master/utils/import-caffe.py

I thought I check here to see if there's already a script for CCV, and whether there's interest in creating one if not.

Gerald

**Comment by liuliu** [comment]
No, not yet. I am expecting some contributions as I am distracted on other tasks for 0.7 / 0.8 release. It would look like this script (in reverse fashion: https://github.com/petewarden/ccv/commit/636531e9b1b6b3b250add0054a99b6e27853bf12).

**Comment by mbertini** [comment]
The latest 0.7 branch has a VGG-D model. Did you create a script to convert those Caffe Zoo models ?

**Comment by liuliu** [comment]
It is trained from scratch. Their model is distributed under CC NC 4 (non
commercial use). I cannot simply redistribute under different license :)

**Comment by ghost** [comment]
@liuliu  any update for parsing the caffemodel to ccv

**Comment by craftzdog** [comment]
+1
